common/djangoapps/third_party_auth/migrations/0012_auto_20230906_2050.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.20 on 2023-09-06 20:50
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sites', '0002_alter_domain_unique'),
+        ('third_party_auth', '0011_applemigrationuseridinfo'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='ltiproviderconfig',
+            name='site',
+            field=models.ForeignKey(default=2, help_text='The Site that this provider configuration belongs to.', on_delete=django.db.models.deletion.CASCADE, related_name='ltiproviderconfigs', to='sites.site'),
+        ),
+        migrations.AlterField(
+            model_name='oauth2providerconfig',
+            name='site',
+            field=models.ForeignKey(default=2, help_text='The Site that this provider configuration belongs to.', on_delete=django.db.models.deletion.CASCADE, related_name='oauth2providerconfigs', to='sites.site'),
+        ),
+        migrations.AlterField(
+            model_name='samlconfiguration',
+            name='site',
+            field=models.ForeignKey(default=2, help_text='The Site that this SAML configuration belongs to.', on_delete=django.db.models.deletion.CASCADE, related_name='samlconfigurations', to='sites.site'),
+        ),
+        migrations.AlterField(
+            model_name='samlproviderconfig',
+            name='site',
+            field=models.ForeignKey(default=2, help_text='The Site that this provider configuration belongs to.', on_delete=django.db.models.deletion.CASCADE, related_name='samlproviderconfigs', to='sites.site'),
+        ),
+    ]
```

Comment:
@viadanna, why is this migration here?

Reply:
@Agrendalath quality pointed out missing migrations. I removed the commit anyway.